eng/Dependencies.props

```diff
@@ -167,6 +167,8 @@ and are generated based on the last package release.
     <LatestPackageReference Include="MessagePack" />
     <LatestPackageReference Include="MessagePackAnalyzer" />
     <LatestPackageReference Include="Microsoft.Data.SqlClient" />
+    <LatestPackageReference Include="Microsoft.Identity.Web" />
+    <LatestPackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" />
```

Comment:
Why are these needed❔ I don't see `@(Reference)` items using either.

Reply:
Ah force of habit. Right these are only used in the template